sessions/react-state-2/conditional-usestate/README.md

```diff
@@ -0,0 +1,30 @@
+# React State 2: Conditional useState
+
+Click the button "Show Message" to render a hidden message to the UI. The App will crash because there is a bug in the code. You won't see the hidden message, but an eslint error regarding the `useState` hook being conditionally rendered.
```

Comment:
After reading it twice: it's actually not an eslint error. The eslint error is displayed in the editor. This is an error message provided by react beacuse the app crashed.

```suggestion
Click the button "Show Message" to render a hidden message to the UI. The App will crash because there is a bug in the code. You won't see the hidden message, but an error regarding the `useState` hook being conditionally rendered.
```